mappings/net/minecraft/JsonSerializableType.mapping

```diff
@@ -0,0 +1,3 @@
+CLASS net/minecraft/class_5336 net/minecraft/JsonSerializableType
```

Comment:
1. Should be `net/minecraft/util/JsonSerializableType`
2. What's the difference between this and `JsonSerializable`?

Reply:
Yes, should be in util.

Its an odd class, as its just holds an instance of JsonSerializable, wasnt too sure what to go with here.